examples/lora/lorawan/basic-demo/main.go

```diff
@@ -81,14 +83,31 @@ func main() {
 	// Connect the lorawan with the Lora Radio device.
 	lorawan.UseRadio(radio)
 
+	lorawan.AddRegionSet("eu868", region.NewRegionSetEU868())
```

Comment:
If you only had a single region, it would be great to be able to just have a one-liner like:

```go
lorawan.UseRegionSet(region.EU868())
```

See my other comment about renaming the function.

Reply:
Or maybe `UseRegionSettings(region.EU868()`?